WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -231,7 +231,11 @@ def create
       registration_events_attributes: [:id, :event_id, :_destroy],
     ]
     if current_user.can_manage_competition?(competition_from_params)
-      permitted_params << :status
+      permitted_params << :accepted_at
+      status = params[:registration][:status]
+      if status
```

Comment:
Ahh, I understand now. I didn't realize you would have to change the tests because they're not using `:a`. What you've got right now looks good to me.